exchanges/websocket/wshandler/wshandler.go

```diff
@@ -656,32 +655,82 @@ func (w *WebsocketConnection) Dial(dialer *websocket.Dialer, headers http.Header
 	return nil
 }
 
-// SendMessage the one true message request. Sends message to WS
-func (w *WebsocketConnection) SendMessage(data interface{}) error {
+// SendJSONMessage sends a JSON encoded message over the connection
+func (w *WebsocketConnection) SendJSONMessage(data interface{}) error {
 	w.Lock()
 	defer w.Unlock()
 	if !w.IsConnected() {
 		return fmt.Errorf("%v cannot send message to a disconnected websocket", w.ExchangeName)
 	}
-	json, err := json.Marshal(data)
-	if err != nil {
-		return err
+	if w.Verbose {
+		log.Debugf(log.WebsocketMgr,
+			"%v sending message to websocket %v", w.ExchangeName, data)
```

Comment:
Please use %+v for data

Reply:
woah man, I never knew about that plus flag. 🎉 🎉 🎉 🎉 🎉